test/document.spec.js

```diff
@@ -546,22 +547,52 @@ describe('Document', () => {
 
   describe('Lock Status', () => {
     describe('#lock', () => {
-      it('should lock the document', () => (
+      const lockDocument = () => {
         repository.fetch(FILE_TEST_PATH)
           .then((doc) => doc.lock())
           .then((doc) => doc.fetchLockStatus())
           .then((status) => {
             expect(status.lockOwner).to.exist();
             expect(status.lockCreated).to.exist();
-          })
-      ));
+          });
+      };
+
+      it('should lock the document', lockDocument);
 
-      /* eslint prefer-arrow-callback: 0 */
       it('should throw an error when locking a document already locked', function f(done) {
+        if (nuxeo.serverVersion.gt(LTS_2019)) {
+          this.skip();
+        }
+
         repository.fetch(FILE_TEST_PATH)
           .then((doc) => doc.lock())
           .then(() => done('Must not resolved when locking a document already locked'), () => done());
       });
+
+      it('should lock the locked document if the user is the lock owner', function f() {
```

Comment:
Add the `done` argument here, `function f(done)`.

When you're using `function f()` instead of an arrow function, we need to use `done` to validate an asynchronous call (like other tests). When using an arrow function, we can just return the promise object.